src/libsync/networkjobs.cpp

```diff
@@ -150,131 +150,171 @@ bool MkColJob::finished()
     return true;
 }
 
-/*********************************************************************************************/
-// supposed to read <D:collection> when pointing to <D:resourcetype><D:collection></D:resourcetype>..
-static QString readContentsAsString(QXmlStreamReader &reader)
-{
-    QString result;
-    int level = 0;
-    do {
-        QXmlStreamReader::TokenType type = reader.readNext();
-        if (type == QXmlStreamReader::StartElement) {
-            level++;
-            result += "<" + reader.name().toString() + ">";
-        } else if (type == QXmlStreamReader::Characters) {
-            result += reader.text();
-        } else if (type == QXmlStreamReader::EndElement) {
-            level--;
-            if (level < 0) {
-                break;
-            }
-            result += "</" + reader.name().toString() + ">";
-        }
 
-    } while (!reader.atEnd());
-    return result;
+LsColXMLParser::LsColXMLParser()
+{
+    currentPropsHaveHttp200 = false;
```

Comment:
i'd say it is better to initialize all of them in the header, then we do not forget to initialize some